integrations/fake-integration/.env.example

```diff
@@ -0,0 +1,2 @@
+OCEAN__PORT__CLIENT_ID="<port-client-id>"
+OCEAN__PORT__CLIENT_SECRET="<port-client-secret>"
```

Comment:
add `base_url` as well?

Reply:
Interesting, some of our integrations have it in the `.env.example`, others don't.